priv/signs.json

```diff
@@ -8375,6 +8375,80 @@
     "chelsea_bridge": "audio",
     "max_minutes": 30
   },
+  {
+    "id": "bus.Nubian_Platform_A",
+    "pa_ess_loc": "SDUD",
+    "read_loop_interval": 420,
+    "read_loop_offset": 180,
+    "text_zone": "w",
+    "audio_zones": [
+      "c"
```

Comment:
Just double checking that these two platforms share the `c` audio zone?

Reply:
It looks like they do, yes. The `read_loop_offset` is different, so the readouts will be staggered.